cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -75,6 +74,8 @@ class LeadImpl extends ServerImpl with Lead
   private var remoteInterpreterServerClass: Class[_] = _
   private var remoteInterpreterServerObj: Any = _
 
+  var urlclassloader: ExtendibleURLClassLoader = _
```

Comment:
Can it be made a val? We are just adding urls to it and not changing the reference as such.

Reply:
May be lazy val.